cmd/crc/cmd/status.go

```diff
@@ -90,6 +90,14 @@ func runWatchStatus(writer io.Writer, client *daemonclient.Client, cacheDir stri
 			if err != nil {
 				return
 			}
+		} else if len(loadResult.CPUUse) > len(cpuBars) {
```

Comment:
should we also consider the case when `loadResult.CPUUse` would be less then `cpuBars` which would occur if the user sets config option `cpus` with a value less then current, currently we have a _stale_ progress bar in this case.